ci/toolchains/TC-arm-unknown-linux-gnueabi.cmake

```diff
@@ -1,5 +1,4 @@
 set(CMAKE_SYSTEM_NAME Linux)
 set(CMAKE_SYSTEM_PROCESSOR arm)
 set(ZENOHC_CUSTOM_TARGET arm-unknown-linux-gnueabi)
-set(CMAKE_C_COMPILER arm-linux-gnueabi-gcc)
-set(CMAKE_CXX_COMPILER arm-linux-gnueabi-g++)
+set(CMAKE_C_COMPILER arm-linux-gnueabi-gcc)
```

Comment:
we likely still need a c++ compiler entry, to prevent cmake from complaining that it was not specified

Reply:
Added